src/popup/pages/Names/AuctionBid.vue

```diff
@@ -25,6 +26,7 @@
           </template>
         </DetailsItem>
       </div>
+
       <BtnMain
         :disabled="!!amountError || error || !amount"
```

Comment:
```suggestion
        :disabled="amountError || error || !amount"
```

Reply:
amountError is not a boolean - it is a `null` or `TranslateResult`. The prop requires to be boolean.